ihmc-common-walking-control-modules/src/test/java/us/ihmc/commonWalkingControlModules/momentumBasedController/optimization/DynamicsMatrixCalculatorTest.java

```diff
@@ -358,9 +362,13 @@ private void setupTest()
       wrenchMatrixCalculator = toolbox.getWrenchMatrixCalculator();
       jointIndexHandler = toolbox.getJointIndexHandler();
 
+      for (ContactablePlaneBody contactablePlaneBody : toolbox.getContactablePlaneBodies())
+         wrenchMatrixCalculator.submitPlaneContactStateCommand(nextPlaneContactStateCommand(random, contactablePlaneBody));
```

Comment:
Without this, all the propagation of the wrench forces were zero. That means that the handling of contact forces between the inverse dynamics calculator and the dynamics matrix calculators weren't actually being tested. This then means that the bug on the sign of the contact forces wasn't caught. Adding this in caused all the tests to fail, and then pass again when the sign was fixed.